merlin-driver/src/main/java/gov/nasa/jpl/aerie/merlin/driver/engine/SimulationEngine.java

```diff
@@ -398,14 +400,23 @@ public SimulationResults computeResults(
       }
     }
 
-    engine.tasks.forEach((task, state) -> {
+    final var usedActivityInstanceIds =
+        taskToPlannedDirective
+            .values()
+            .stream()
+            .map(ActivityInstanceId::id)
+            .collect(Collectors.toSet());
+    var counter = 1L;
```

Comment:
(continuing from the previous thread) @kroffo is there a reason why this isn't `long counter = 0;`?

Reply:
`var`: Because `var` is superior to explicit types. If we for some reason change the value, the type declaration needn't change.

`1`: The value 1 was chosen because 1 is what we use when doing this same thing in the `PostgresResultsCellRepository`, though it doesn't actually matter. I also thought about it and thought that if a plan starts its activity IDs from 1, it would feel odd to have one child activity at ID 0 and then the rest at higher numbers after planned activities. Of course, again, it doesn't actually matter because the numbers are still unique, but those were my thoughts.

Reply:
Ahh I see, thanks for explaining!

Reply:
Arguably, we should start from Long.MIN_VALUE; otherwise we're wasting half the ID space!

No, I'm kidding. I'd argue for starting from 0 in general. Your reasoning is sound, but it makes me wonder whether the `PostgresResultsCellRepository` can be changed to start from 0 too.